packages/ts/src/components/donut/config.ts

```diff
@@ -62,4 +68,6 @@ export const DonutDefaultConfig: DonutConfigInterface<unknown> = {
   emptySegmentAngle: 0.5 * Math.PI / 180,
   showBackground: true,
   backgroundAngleRange: undefined,
+  centralLabelsOffsetX: undefined,
```

Comment:
For default value, use `0` here.

Reply:
@jakubjosef I had it 0 originally but changed the default to `undefined` as suggested by @rokotyan here https://github.com/ExaForce/unovis/pull/9#discussion_r1878579701